src/raftstore/store/raft_engine_metrics.rs

```diff
@@ -0,0 +1,285 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+use prometheus::{Gauge, GaugeVec};
+use rocksdb::{DB, DBStatisticsTickerType as TickerType, DBStatisticsHistogramType as HistType,
+              HistogramData};
+use storage::CF_DEFAULT;
+use super::CF_RAFT;
+use util::rocksdb;
+use super::engine_metrics::*;
+
+pub fn flush_raft_engine_ticker_metrics(t: TickerType, value: u64) {
```

Comment:
can we use the metric with the first label is engine like `metrics.with_label_values(&["kv", "a"])` and `metrics.with_label_values(&["raft", "a"])` to avoid these duplicated codes.